src/screens/collectibles-home/collectibles-home.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import { Divider } from '../../components/divider/divider';
+import { HeaderCard } from '../../components/header-card/header-card';
+import { Icon } from '../../components/icon/icon';
+import { IconNameEnum } from '../../components/icon/icon-name.enum';
+import { ScreenContainer } from '../../components/screen-container/screen-container';
+import { SearchInput } from '../../components/search-input/search-input';
+import { useFilteredAssetsList } from '../../hooks/use-filtered-assets-list.hook';
+import { ScreensEnum } from '../../navigator/enums/screens.enum';
+import { useNavigation } from '../../navigator/hooks/use-navigation.hook';
+import { useVisibleCollectiblesListSelector } from '../../store/wallet/wallet-selectors';
+import { formatSize } from '../../styles/format-size';
+import { useCollectiblesHomeStyles } from './collectibles-home.styles';
+import { CollectiblesList } from './collectibles-list/collectibles-list';
+import { PromotionCarousel } from './promotion-carousel/promotion-carousel';
+
+export const CollectiblesHome = () => {
+  const styles = useCollectiblesHomeStyles();
+  const { navigate } = useNavigation();
+
+  const visibleCollectiblesList = useVisibleCollectiblesListSelector();
+  const { filteredAssetsList, setSearchValue } = useFilteredAssetsList(visibleCollectiblesList);
+
+  const [isSearchMode, setIsSearchMode] = useState(false);
```

Comment:
lint ignore?

Reply:
let this warning live here.  
this will be used with the uncommented search below.